src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -958,6 +1134,51 @@ def forward(
         output_hidden_states=None,
         return_dict=None,
     ):
+        r"""
+        labels (:obj:`torch.LongTensor` of shape :obj:`(batch_size, sequence_length)`, `optional`):
+            Labels for computing the token classification loss. Indices should be in ``[0, ..., config.num_labels -
+            1]``.
+
+        Returns:
+
+        Examples::
+
+            >>> from transformers import LayoutLMTokenizer, LayoutLMForTokenClassification
+            >>> import torch
+
+            >>> tokenizer = LayoutLMTokenizer.from_pretrained('microsoft/layoutlm-base-uncased')
+            >>> model = LayoutLMForTokenClassification.from_pretrained('microsoft/layoutlm-base-uncased')
+
+            >>> words = ["Hello", "world"]
+            >>> normalized_word_boxes = [637, 773, 693, 782], [698, 773, 733, 782]
+            >>> inputs = tokenizer(words, return_tensors="pt")
+            >>> input_ids = inputs['input_ids']
+            >>> attention_mask = inputs['attention_mask']
+            >>> token_type_ids = inputs['token_type_ids']
+
+            >>> tokens = []
+            >>> token_boxes = []
+            >>> for word, box in zip(words, normalized_word_boxes):
```

Comment:
Same as previous, think it should be `>>>` below. Also, it would be great if we could just use `tokenizer(...)` instead of `tokenize(...)` and `convert_tokens_to_ids`. Is the problem that we need to know the length of each sample and this doesn't work if the input is batched? If it's the case I think it'd be better to compute `token_boxes` separately and try to just use one function call for `input_ids`

Reply:
Can't see with the comment, but there might be a space missing there too ;-)